File: Models/Responses/Process/BusinessUnit/ResponseProcessBusinessUnitSummary.cs
namespace Models.Responses.Process.BusinessUnit;

/// <summary>
/// 결과중 개별 Owner 별 통계 데이터 모음
/// </summary>
public class ResponseProcessBusinessUnitSummary
{
    /// <summary>
    /// 오너정보
    /// </summary>
    public List<ResponseProcessBusinessUnitSummaryDetail> Items { get; set; } = new List<ResponseProcessBusinessUnitSummaryDetail>();
}

File: BudgetBringerV2/src/models/responses/process/process-business-unit/response-process-business-unit-summary.ts
import {ResponseProcessBusinessUnitSummaryDetail} from "./response-process-business-unit-summary-detail";

/**
 * 결과중 개별 ProcessOwner 별 통계 데이터 모음
 */
export class ResponseProcessBusinessUnitSummary {
  /**
   * 오너정보
   */
  items: ResponseProcessBusinessUnitSummaryDetail[];
}
